### PR TITLE
Add Florian Verger

### PR DESCRIPTION
**Pull request by vergerf**

### CODE REVIEW

File: baggers.js [hunk 1]
```diff
@@ -13616,6 +13616,48 @@ var data = {
...
      }
+    },
+    {
+      "since": "2016-09-29",
+      "name": "Florian Verger",
+      "bio": "Ingénieur R&D chez Lucca, éditeur de logiciels SaaS de gestion RH.",
+      "picture": "https://secure.gravatar.com/avatar/a896cbaf08bb15196c309e8055e0b814?size=200",
+      "websites": [
+        {
+          "name": "lucca",
```
Comment (dadoonet) [inline baggers.js]: Please use the `name` allowed in the README file: https://github.com/brownbaglunch/bblfr_data#websites

File: baggers.js [hunk 2]
```diff
@@ -13616,6 +13616,48 @@ var data = {
...
+          "url": "https://www.linkedin.com/in/florian-verger-156a8a67"
+        }
+      ],
+      "location": "Nantes",
+      "sessions": [
+        {
+          "title": "Réécrivez l'histoire avec git",
+          "abstract": "Présentation sur des cas concrets afin de garder un historique git propre et lisible. Fini le commit 'fix de fix'",
+          "tags": [
+            "git",
```
Comment (dadoonet) [inline baggers.js:13681]: Just an opinion here but `git` as a tag should be enough. Plus `sourcetree`. Feel free to ignore my comment though.

**Review by dadoonet — CHANGES_REQUESTED** [review]
Could you rebase on master and apply the comments I sent please?
Thanks!

**Comment by vergerf** [comment]
I've made all changes you requested.

**Review by dadoonet — APPROVED** [review]

**Comment by dadoonet** [comment]
Bienvenue !

**Pull request merged**